src/components/GlobalNavbar/GlobalNavbar.tsx

```diff
@@ -1,12 +1,15 @@
+import { useAuthState } from 'contexts/auth/AuthContext';
 import styled from 'styled-components';
-import SigninButton from './SigninButton';
+import SignInButton from './SignInButton';
+import SignOutButton from './SignOutButton';
 
 function GlobalNavbar() {
+  const authState = useAuthState();
+  console.log(typeof authState);
+  const isLoggedIn = typeof authState === 'object';
```

Comment:
we should probably write a [typeguard](https://basarat.gitbook.io/typescript/type-system/typeguard) for this, which will be a function like `function isLoggedIn(state: authState) ...` and returns a boolean

Reply:
function could live in contexts/auth/types.ts

Reply:
done